core/controllers/moderator.py

```diff
@@ -49,3 +50,15 @@ def post(self):
         self.render_json({
             'user_email': user_services.get_email_from_username(username),
         })
+
+
+class EmailDraftHandler(base.BaseHandler):
```

Comment:
It seems strange that this is supplying the content but not the subject; I feel the two should be treated equivalently throughout the code.